src/sdk/object_sdk.js

```diff
@@ -307,6 +304,7 @@ class ObjectSDK {
 
     _setup_merge_namespace(bucket) {
         let rr = _.cloneDeep(bucket.namespace.read_resources);
+        /** @type {nb.Namespace} */
```

Comment:
This `@type` is for the `wr` variable?
If so, why not add the `@type` line before `rr` variable statement?

Reply:
leaving something for others :)